code/modules/admin/admin_verbs.dm

```diff
@@ -233,7 +234,16 @@ var/list/admin_verbs_permissions = list(
 	/client/proc/drop_bomb
 	)
 var/list/admin_verbs_rejuv = list(
-	/client/proc/respawn_character
+	/client/proc/respawn_character,
+	/client/proc/cmd_admin_rejuvenate,
+	/datum/admins/proc/announce,		//priority announce something to all clients.,
+	/datum/admins/proc/restart,
+	/datum/admins/proc/delay,
+	/datum/admins/proc/toggleooc,		//toggles ooc on/off for everyone,
+	/datum/admins/proc/toggleaooc,		//toggles aooc on/off for everyone,
+	/datum/admins/proc/togglelooc,		//toggles looc on/off for everyone,
+	/datum/admins/proc/toggleoocdead,	//toggles ooc on/off for everyone who is dead,
+	/datum/admins/proc/toggledsay		//toggles dsay on/off for everyone,
```

Comment:
these various communicate procs don't have much to do with rejuvenating, why not just give them to moderators instead?

Reply:
+MOD or R_MOD in this case is handed out to regular moderators, including trial moderators. This current change puts it squarely under Senior Mod permissions. Prior, admins had this permission as only one. Rather than a three-rank downward bump on permissions, I just made it singular. Better odds of getting it past the headmins.